core/os/spin_lock.h

```diff
@@ -37,12 +37,26 @@
 
 #include <os/lock.h>
 
+#if defined(__aarch64__) && defined(MACOS_ENABLED)
+#define OS_UNFAIR_LOCK_DATA_SYNCHRONIZATION (0x00010000)
+#define OS_UNFAIR_LOCK_ADAPTIVE_SPIN (0x00040000)
```

Comment:
We're preferring first-class constants where possible;
```suggestion
static constexpr uint32_t OS_UNFAIR_LOCK_DATA_SYNCHRONIZATION = 0x00010000;
static constexpr uint32_t OS_UNFAIR_LOCK_ADAPTIVE_SPIN = 0x00040000;
```

Would there be any issue in this case?